src/frontend/src/pages/DesignReviewView.tsx

```diff
@@ -0,0 +1,186 @@
+              const index = dayIndex * times.length + timeIndex;
+              return (
+                <TimeSlot
+                  key={index}
+                  backgroundColor={getBackgroundColor(availableUsers.get(index)?.length, totalUsers)}
+                  onMouseOver={() => handleOnMouseOver(index)}
+                  isModal={false}
+                  icon={iconData.get(index)}
+                />
+              );
+            })}
+          </Grid>
+        ))}
+      </Grid>
+    );
+  };
+
+  const renderLegend = () => {
+    const colors = ['#D9D9D9', '#E0C0C1', '#E89A9B', '#E4797A', '#EF4345', '#D70C0F'];
+    return (
+      <Grid sx={{ display: 'flex', justifyContent: 'center' }}>
+        <Typography style={{ marginRight: '10px', fontFamily: 'oswald' }}>0/0</Typography>
+        {Array.from({ length: 6 }, (_, i) => (
+          <Box
+            sx={{
+              width: '1.5vw',
+              height: '1.5vw',
+              backgroundColor: colors[i]
+            }}
+          />
+        ))}
+        <Typography style={{ marginLeft: '10px', fontFamily: 'oswald' }}>
+          {totalUsers}/{totalUsers}
+        </Typography>
+      </Grid>
+    );
+  };
+
+  const renderAvailabilites = () => {
+    const fontSize = totalUsers > 10 ? '1em' : totalUsers > 15 ? '0.8em' : '1.2em';
+    return (
+      <Grid
+        style={{
+          display: 'flex',
+          flexDirection: 'column',
+          backgroundColor: '#2C2C2C',
+          padding: '20px',
+          borderRadius: '8px',
+          height: '100%',
+          overflow: 'auto'
+        }}
+      >
+        <Grid style={{ display: 'flex', flexDirection: 'column', flexGrow: 1, marginBottom: '10px' }}>
+          {renderLegend()}
+          <Grid style={{ display: 'flex', gap: '2', marginTop: '10px' }}>
+            <Grid>
+              <Typography
+                style={{
+                  textDecoration: 'underline',
+                  fontFamily: 'oswald',
+                  fontSize: '1.5em',
+                  textAlign: 'center',
+                  marginBottom: '10px'
+                }}
+              >
+                Available Users
+              </Typography>
+              {currentAvailableUsers.map((user) => (
+                <Typography style={{ fontFamily: 'oswald', textAlign: 'center', fontSize }}>
+                  {user.firstName} {user.lastName}
+                </Typography>
+              ))}
+            </Grid>
+            <Grid>
+              <Typography
+                style={{
+                  textDecoration: 'underline',
+                  fontFamily: 'oswald',
+                  fontSize: '1.5em',
+                  textAlign: 'center',
+                  marginBottom: '10px'
+                }}
+              >
+                Unvailable Users
+              </Typography>
+              {currentUnavailableUsers.map((user) => (
+                <Typography style={{ fontFamily: 'oswald', textAlign: 'center', fontSize }}>
```

Comment:
you shouldn't ever need to set a fontFamily we configure that at an app level. If it looks super weird as the default font, lmk